index.html

```diff
@@ -3524,4 +3484,5 @@ <h2 id="changes-from-fpwd">Changes from the FPWD published 24 November 2020</h2>
     </ul>
   </section>
 </body>
-</html>
+
+</html>
```

Comment:
If you happen to need to rework it, please add a newline here to make git happier, otherwise it is fine.